sharding-scaling/sharding-scaling-core/src/main/java/org/apache/shardingsphere/shardingscaling/core/execute/executor/channel/MemoryChannel.java

```diff
@@ -35,9 +34,9 @@
  */
 public class MemoryChannel implements Channel {
 
-    private static final int PUSH_TIMEOUT = ScalingContext.getInstance().getServerConfiguration().getPushTimeout();
+    private static final int PUSH_TIMEOUT = 1000;
 
-    private final BlockingQueue<Record> queue = new ArrayBlockingQueue<>(ScalingContext.getInstance().getServerConfiguration().getBlockQueueSize());
+    private final BlockingQueue<Record> queue = new ArrayBlockingQueue<>(10000);
```

Comment:
Why do you use hardcode in there?

Reply:
I will submit a new issue for config related.